rest/utilities_testing.go

```diff
@@ -2584,6 +2579,17 @@ func (sc *ServerContext) RequireInvalidDatabaseConfigNames(t *testing.T, expecte
 	require.EqualValues(t, expectedDbNames, dbNames)
 }
 
+// AllInvalidDatabaseNames returns the names of all the databases that have invalid configs. Testing only since this locks the database context.
+func (sc *ServerContext) AllInvalidDatabaseNames(_ *testing.T) []string {
```

Comment:
It looks like every test use of AllInvalidDatabaseNames is actually only interested in the length of invalidDatabaseConfigTracking.dbNames.  Would a helper function for length only make more sense?

Reply:
I'm inclined to leave this - the places this is called is for checking the length, but we can't directly do:

`require.Len(t, sc.AllInvalidDatabaseNames(), 1)`

because these are done in retry loops. The advantage of having the list of names is that testify's `require.Len` will report something like: ["dbA", "dbB"] is not of len 1 - so you see the names in the error message.